Cartfile.resolved

```diff
@@ -2,7 +2,7 @@ binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" "119.0.2"
 github "Quick/Nimble" "v9.2.1"
 github "Quick/Quick" "v3.1.2"
-github "mapbox/mapbox-directions-swift" "v2.8.0-rc.2"
+github "mapbox/mapbox-directions-swift" "v2.8.0"
 github "mapbox/mapbox-events-ios" "v1.0.10"
```

Comment:
Should there also be 1.0.8? Carthage project doesn't require Maps so `mapbox-events-ios` is not downgraded herw, but I think we can get something unexpected if we build with 1.0.10 in one project and with 1.0.8 in another.

Reply:
Ok, turns out I needed to call "Update to Latest Package Versions" in Xcode to align all the dependencies. Now SPM prefers 1.0.10 too.